storage/stdin.go

```diff
@@ -0,0 +1,26 @@
+package storage
+
+import (
+	"io"
+	"os"
+)
+
+type StdinReader interface {
```

Comment:
Do we really need `io.ReadCloser` wrapper and `storage.StdinReader` abstraction for `os.Stdin`? It is used as `io.Reader` in pipe command anyways as `storage.Storage.Put` interface accepts an `io.Reader`. Par with `cat` command, I think we can use `os.Stdin` as is;
https://github.com/peak/s5cmd/blob/74d21bb4f022edb0bfb69f412c4baa69153ba8c2/command/cat.go#L106

Reply:
Yes, you're right. I simplified it but still we can't use `os.stdin` directly due to limitations in AWS SDK (it throws illegal seek error).
https://github.com/peak/s5cmd/pull/587/commits/19d2675fe8f27d9f92ebfb5fda5aeb1e16cc3d91